### DISCUSSION TITLE
need help I am been blackmailed by spyware and malware hacker

### DISCUSSION THREAD

**Discussion by anonymouslyblack** [mvt-project/mvt#323] [General]
hello please I need help someone hacked my device and uploaded spywares and I don't know how according to his threats he has been watching me for a very long time from my camera he is saying he would upload some bad videos of me unless I pay money to him please I need help

**Comment by Te-k** [comment]
Hi, these messages are classic scams, you should ignore them.

**Comment by AndreasMccann** [comment]
I'm sorry to hear about your situation. The first thing you should do is not to pay the person who is threatening you. This will only encourage them to continue their criminal activities and will not guarantee that they will delete any compromising content they claim to have. You have fallen victim to [online sextortion](https://www.digitalforensics.com/blackmail), which is a very serious and damaging crime.  You should also contact law enforcement immediately and report the incident. They can work with you to investigate and hopefully identify the perpetrator.
